src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -89,6 +89,15 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * int[] array; // OK
+ * int [] array; // violation, whitespace between int and opening bracket [
```

Comment:
please combine all example in single java code (it will be a bit big) and use this block for each config.
java code should not change expect for trailng comments like "ok" and "violation".